public/js/main.js

```diff
@@ -268,6 +283,17 @@ document.addEventListener("DOMContentLoaded", () => {
 
       buildOrglist();
       initListJS();
+
+      const configArray = JSON.parse(localStorage.getItem("config"));
+      if (configArray !== null || configArray.length !== 0) {
```

Comment:
It throws
> main.js:288 Uncaught TypeError: Cannot read properties of null (reading 'length')
    at WebSocket.<anonymous>
 
if config is empty.
`if (configArray?.length > 0)`should do the job